setup.py

```diff
@@ -38,12 +38,35 @@ def read_version():
     "numpy>=1.9.0",
     "protobuf>=3.1",
     "scipy>=0.19.0",
-    "urllib3>=1.21, <1.25",
     "protobuf3-to-dict>=0.1.5",
     "requests>=2.20.0, <2.21",
     "fabric>=2.0",
```

Comment:
I realize this was part of an old change, but `fabric` should actually be part of test requires

Reply:
Thanks!